docs/source/concepts/recipe.rst

```diff
@@ -182,35 +190,25 @@ Output metadata
 After the package is built and tested, conda-build cleans up the
 environments created during prior steps and outputs the metadata. The recipe for
 the package is also added in the output metadata. The metadata directory
-is on the top level of the tarball in the ``info`` directory.
+is at the top level of the package contents in the ``info`` directory.
 The metadata contains information about the dependencies of the
 package and a list of where all of the files in the package go when
 it is installed. Conda reads that metadata when it needs to install.
 
 Running ``conda install`` causes conda to:
 
-  * reach out to the repo data containing the dependencies,
-  * guess the right dependencies,
-  * install a list of packages,
-  * unpack the tarball to look at the info,
-  * verify the file based on metadata in the package, and then
-  * go through each file in the package and put it in the right location.
-
-
-More information
-================
-
-Review :doc:`../resources/define-metadata` to see a breakdown of the
-components of a recipe, including:
-
-  * Package name
-  * Package version
-  * Descriptive metadata
-  * Where to obtain source code
-  * How to test the package
+#. Reach out to the repodata containing the dependencies for the package(s) you are installing.
+#. Determine the correct dependencies.
+#. Install a list of additional packages determined by those dependencies.
+#. For each dependency package being installed:
+   #. Unpack the tarball to look at the information contained within.
```

Comment:
Wasn't sure about this, but was assuming that the last three things are done for each package, yes?